### PR TITLE
[TUBEMQ-530] create a cluster in manager

### PR DESCRIPTION
**Pull request by EMsnap**
[TUBEMQ-530] create a cluster in manager

### CODE REVIEW

File: tubemq-manager/pom.xml [hunk 1]
```diff
@@ -54,6 +54,12 @@
             <version>4.3</version>
         </dependency>
 
+        <dependency>
```
Comment (yuanboliu) [inline tubemq-manager/pom.xml:57]: seems useless in this patch

File: tubemq-manager/src/main/java/org/apache/tubemq/manager/entry/NodeEntry.java [hunk 2]
```diff
@@ -32,8 +32,8 @@
 @Data
 public class NodeEntry {
     @Id
-    @GeneratedValue(strategy= GenerationType.AUTO)
-    private long brokerId;
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
```
Comment (yuanboliu) [inline tubemq-manager/src/main/java/org/apache/tubemq/manager/entry/NodeEntry.java:35]: what if clusters have conflicted broker ids. 
for example A cluster has broker which id is num_a, meanwhile B cluster also has broker with the same id.
It seems better that we need create a colume called brokerId without any annotation.

File: tubemq-manager/src/main/java/org/apache/tubemq/manager/entry/ClusterEntry.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ */
+@Entity
+@Table(name = "cluster")
+@Data
+public class ClusterEntry {
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private int clusterId;
+
+    private String clusterName;
```
Comment (yuanboliu) [inline tubemq-manager/src/main/java/org/apache/tubemq/manager/entry/ClusterEntry.java:40]: cluster name should be marked as uniq key.

File: tubemq-manager/pom.xml (hunk 1 above)
Comment (EMsnap) [reply tubemq-manager/pom.xml:57]: 21.0

**Comment by yuanboliu** [comment]
+1

**Pull request merged**